### PR TITLE
Follow UP Pr to adding a GQL warning for not using Graphql while using Cody API 

### PR DESCRIPTION
**Pull request by arafatkatze**
Coming from https://github.com/sourcegraph/sourcegraph/pull/61381#issuecomment-2021634637

image

## Test plan
When you run a local instance and use the completions endpoint you will see a yellowed out sign at the bottom and a warning

### CODE REVIEW

File: cmd/frontend/graphqlbackend/completions.graphql [hunk 1]
```diff
@@ -3,6 +3,9 @@ extend type Query {
     Returns a string of completion responses
     """
     completions(input: CompletionsInput!, fast: Boolean = false): String!
+        @deprecated(
+            reason: "Consumers should use non graphql endpoints when communicating the completions endpoint. Those endpoints can be found in this file -> https://sourcegraph.com/github.com/sourcegraph/cody/-/blob/vscode/src/services/AuthProvider.ts"
```
Comment (slimsag) [inline cmd/frontend/graphqlbackend/completions.graphql:7]: Can you link to (or create if needed) a page on https://sourcegraph.com/docs that explains with e.g. curl examples how to use the Cody API? That would be the best-case scenario here

Afraid no site admin would know what to do with that typescript file.

Comment (slimsag) [reply cmd/frontend/graphqlbackend/completions.graphql:7]: (just an idea, otherwise not sure the motivation for linking the typescript file)

Comment (arafatkatze) [reply cmd/frontend/graphqlbackend/completions.graphql]: image

Comment (slimsag) [reply cmd/frontend/graphqlbackend/completions.graphql]: okay, sounds fine then

**Review by taras-yemets — APPROVED** [review]

**Comment by arafatkatze** [comment]
Merged already in another PR

**Pull request closed without merging**